src/openforms/config/constants.py

```diff
@@ -1,6 +1,11 @@
 from django.db import models
 from django.utils.translation import gettext_lazy as _
 
+ADDITIONAL_CSP_VALUES = {
+    "digid": "https://digid.nl https://*.digid.nl",
+    "eherkenning": "",
+}
```

Comment:
please move this to `openforms.contrib.digid_eherkenning`

In general, the `config` app is set up in a generic way and consults the various plugin registries, so hardcoding plugin-specific things outside of the plugin are a code smell - it introduces (more) tight coupling than needed.